core/arch/arm/tee/svc_cache.c

```diff
@@ -60,5 +59,5 @@ TEE_Result syscall_cache_operation(void *va, size_t len,
 	if (res != TEE_SUCCESS)
 		return TEE_ERROR_ACCESS_DENIED;
 
-	return cache_operation(op, va, len);
+	return cache_operation((enum utee_cache_operation)op, va, len);
```

Comment:
It is cosmetic. Could it help compiler in some way ? Keeping or removing this cast is a matter of taste ? (i feel confortable with either cases)